src/resources/Pipelines/ResultRankings/ResultRankingsInterfaces.ts

```diff
@@ -13,6 +15,12 @@ export interface ResultRanking {
     associatedGroup?: ResultRankingAssociatedGroup;
 }
 
+export interface ListResultRanking {
+    id: string;
+    resultRanking: ResultRankingProps;
+    associatedGroup?: ResultRankingAssociatedGroupWithStatus;
+}
+
```

Comment:
I'm not hardcore on that but:
>  JSDoc annotates each property added in the exported interfaces 

What about that?

Reply:
It's not even documented in our API but I can write a meaningless description 😅

Reply:
Why do you say meaningless ?

Reply:
```typescript
    /**
     * The unique identifier of the result ranking rule.
     */
    id: string;
```

It's just that I don't think it brings a lot of value to write this kind of documentation. But maybe there's a better way to describe this kind of properties

Reply:
> ```ts
>     /**
>      * The unique identifier of the result ranking rule.
>      */
>     id: string;
> ```
> 
> It's just that I don't think it brings a lot of value to write this kind of documentation. But maybe there's a better way to describe this kind of properties

Overall, public APIs should (and, IMHO, must) be documented, both in Swagger/OpenAPI and in the abstraction of the API such as this project.

If you're unsure about your comments etc, you probably should involve someone from the documentation team, they're the specialist on that ;)